crypto/replay/rdb.c

```diff
@@ -100,7 +100,7 @@ srtp_err_status_t srtp_rdb_check(const srtp_rdb_t *rdb, uint32_t p_index)
  */
 srtp_err_status_t srtp_rdb_add_index(srtp_rdb_t *rdb, uint32_t p_index)
 {
-    int delta;
+    unsigned int delta;
 
```

Comment:
This worried me a bit but the nice comment on the line below  ` /* here we *assume* that p_index > rdb->window_start */`  made it all good again. Maybe we should add an actual check instead of just a comment.

Reply:
Good point. Happy to add that error check. Not sure about which error to return though. srtp_err_status_replay_fail?

Reply:
Unfortunately the return value is not currently checked so srtp_err_status_replay_fail should be fine.